code/modules/research/xenobiology/crossbreeding/reproductive.dm

```diff
@@ -42,11 +42,11 @@ Reproductive extracts:
 		cubes_eaten++
 		to_chat(user, "<span class='notice'>You feed [monkeycube] to [src], and it pulses gently.</span>")
 		playsound(src, 'sound/items/eatfood.ogg', 20, 1)
-
+/*
```

Comment:
Considering the lack of room that would be needed and the inability to do any sort of effective expansion, I would argue to not remove this, though we have industrialized versions for monkey cubes so this moreso affects getting reproductive slimes, would be something to consider

Reply:
FIRSTLY. if you want infinite grey cells, you get the industrialized version. and pay fucking plasma for it. otherwise you take the darn tier 4 monkey recycler.

Reply:
Plasma becomes self sufficient with dark purple smiles

Reply:
Idc